code/modules/projectiles/bullet.dm

```diff
@@ -816,19 +817,31 @@ toxic - poisons
 	implanted = null
 	damage = 6
 
+	on_launch(var/obj/projectile/P)
+		..()
+		if(prob(50)) //let's get some variation in here
+			icon_state = "glass2"
+		if(prob(20)) //we want implants, but we only want a few
```

Comment:
pretty big nerf to glassshot, needs a clog entry imo

Reply:
Glass shot didn't originally have any implants, but you do bring up a good point.